addons/event/models/event_tag.py

```diff
@@ -11,21 +11,26 @@ class EventTagCategory(models.Model):
     _description = "Event Tag Category"
     _order = "sequence"
 
+    def _default_sequence(self):
```

Comment:
Small remark on commit message:

> the tags were ordered only by its own sequence
->
> the tags were ordered only by their own sequence

(Same remark applies a bit later in the message).